edgecast/resources/cps/certificate.go

```diff
@@ -106,59 +117,66 @@ func ResourceCertificateCreate(
 
 func ExpandCertificate(
```

Comment:
We are missing unit tests for ExpandCertificate. We can log as a new ticket since this is an existing method if you prefer.